### DISCUSSION TITLE
Is Storage part of the local docker-compose file?

### DISCUSSION THREAD

**Discussion by 0xflotus** [supabase/supabase#1241] [Questions] [answered]
Is it currently possible to use the new Storage API in development environment?

Currently it's not part of the docker-compose file. Right?

https://github.com/supabase/supabase/blob/1c2970d3a48b696a8c06d5c8e485f5296f589050/docker/docker-compose.yml#L1-L81

So how can I use the new Storage API (provided that I have a S3 bucket) in a local environment? 
Or is it only possible for now to use it with supabase platform?

**Comment by inian** [comment]
Hi @0xflotus, I have a [PR for this](https://github.com/supabase/supabase/pull/1007) but it still requires you have a S3 bucket as you mentioned. We have not merged it in yet since it still requires access to S3 and doesn't run locally completely.

**Reply by 0xflotus** [reply]
Thanks, this answer is all I need :)

**Comment by inian** [comment, marked as answer]
Storage is officially supported in the local setup now - https://supabase.io/blog/2021/07/27/storage-beta#using-storage-in-local-development
